content/responses/folder.yml

```diff
@@ -61,6 +61,54 @@ allOf:
           Specifies if this folder is owned by a user outside of the
           authenticated enterprise.
 
+      metadata:
```

Comment:
Can you clarify if metadata for folders is supported in the get folder by ID api?